durations/src/test/java/DurationTests.kt

```diff
@@ -1,13 +1,34 @@
 package com.redspace.durations
 
-import org.amshove.kluent.shouldBeFalse
-import org.amshove.kluent.shouldBeInstanceOf
-import org.amshove.kluent.shouldEqual
+import org.amshove.kluent.*
```

Comment:
Opinions on * imports vs. explicit?

Reply:
I have no opinion, I just let the IDE do whatever it wants